docs/en/programming/python/tutorial-1.md

```diff
@@ -57,7 +57,7 @@ if __name__ == "__main__":
 
 Please save this file under the name [`test.py`](https://raw.githubusercontent.com/ftCommunity/ftcommunity-apps/master/packages/app_tutorial_1/test.py)
 
-This app subclasses the TxtApplication class which was imported from
+This app subclasses the TouchApplication class which was imported from
 TxtStyle. It creates a window labeled "Test", shows that window and
```

Comment:
"imported from TxtStyle" should be changed to "imported from TouchStyle" as well.